gemrb/core/GameScript/GameScript.cpp

```diff
@@ -2491,7 +2491,7 @@ void GameScript::ExecuteAction(Scriptable* Sender, Action* aC)
 
 Trigger* GenerateTrigger(char* String)
 {
-	StringToLower(String);
+	StringToLower(String, String + strlen(String));
```

Comment:
strnlen would be safer, I already see Sonar complaining, but it would be even better to just convert GenerateTrigger to take a std::string.

Reply:
yeah, my thoughts were to kick this toward stringifying GameScript at some point